docs/platform/what-is-tizen/versions/tizen-5-0-m2.md

```diff
@@ -1,7 +1,7 @@
 # Tizen 5.0
 
 [The second milestone (M2) release of Tizen 5.0](../../release-notes/tizen-5-0-m2.md) was announced in October 2018. Tizen has expanded its horizon from smart devices to IoT devices. Currently, Tizen is used for various kinds of smart devices.
-Tizen 5.0 adds more enhancements to the services and provides the opportunity to develop service-centric applications. With the result, the end-users will get a better experience with more enhanced applications. Tizen 5.0 provides APIs such as Tizen Interface Definition Language (TIDL), RPC-port for easy integration of various kinds of services such as Smart-things. Core open sources such as Blink, GStreamer, EFL, Wayland, and so on are upgraded.
+Tizen 5.0 adds more enhancements to the services and provides the opportunity to develop service-centric applications. With the result, the end-users will get a better experience with more enhanced applications. Tizen 5.0 provides APIs such as Tizen Interface Definition Language (TIDL) and RPC-port for easy integration of various kinds of services such as SmartThings. Many core open sources are upgraded in Tizen 5.0.
```

Comment:
@jhchoilee  this sentence seems incomplete

Can we use components.  
Many core open source components are upgraded in Tizen 5.0.